src/Progress/Store.js

```diff
@@ -1,22 +1,10 @@
 import { combineReducers } from 'redux'
 import BaseStore from '../Core/BaseStore'
+import {groupBy} from '../Utils/groupBy'
 
 export const identifier = 'progress'
-
 const baseStore = BaseStore(identifier)
 
-function groupBy(data, key) {
```

Comment:
das ist auf jeden Fall gut, dass die Methode nicht mehr hier ist! So eine allgemeine Methode gehört ja nicht in einen so konkreten Code-Teil wie den Progress-Store. 👍